### PR TITLE
Updated readme to include how to grab existing planet project 

### PR DESCRIPTION
**Pull request by smilegodly**
I was able to download the planet cdot code from the overview using wget
```
wget -r --no-parent https://people.gnome.org/~jdub/bzr/planet/devel/trunk/
```
Fixes: #29

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,15 @@
 A tool for tracking blogs in orbit around Seneca's open source involvement
 
 An initial discussion of the project is available in the [Overview](docs/overview.md) document.
+
+#Get the Code
```
Comment (humphd) [inline README.md]: Space after `#`

Comment (humphd) [reply README.md]: Let's also mention that this is for the legacy code vs. what we're building.

Comment (cindyledev) [reply README.md]: Adding on to @humphd's comment, we should include that the wget command is for Linux/Mac and doesn't work for Windows. I ran this on Fedora but for some reason the --no-parent option didn't do what it was suppose to do so I ran:
# wget -r --no-parent https://people.gnome.org/~jdub/bzr/planet/devel/trunk/
# mv people.gnome.org/~jdub/bzr/planet/devel/trunk cdotplanet
# cd cdotplanet

Comment (smilegodly) [reply README.md]: The requested changes have been made. 
Please LMK if there are any issues @humphd

**Comment by smilegodly** [comment]
Fixed the conflict that I was having

**Pull request merged**